docs/source/rst/examples.rst

```diff
@@ -69,102 +41,114 @@ The following is printed to the terminal:
 
 .. code-block:: text
 
+  --Initialising the Section class...
+  ----completed in 0.038872 seconds---
+
   Mesh Statistics:
-  --2562 nodes
-  --1247 elements
+  --2551 nodes
+  --1242 elements
   --1 region
 
   --Calculating geometric section properties...
-  ----completed in 0.765906 seconds---
+  ----completed in 1.260965 seconds---
+
+  --Assembling 2551x2551 stiffness matrix and load vector...
+  ----completed in 1.481895 seconds---
 
-  --Assembling 2562x2562 stiffness matrix and load vector...
-  ----completed in 1.107300 seconds---
   --Solving for the warping function using the direct solver...
-  ----completed in 0.023138 seconds---
+  ----completed in 0.027327 seconds---
+
   --Computing the torsion constant...
-  ----completed in 0.000254 seconds---
+  ----completed in 0.000417 seconds---
+
   --Assembling shear function load vectors...
-  ----completed in 1.150968 seconds---
+  ----completed in 1.592697 seconds---
+
   --Solving for the shear functions using the direct solver...
-  ----completed in 0.136840 seconds---
+  ----completed in 0.155810 seconds---
+
   --Assembling shear centre and warping moment integrals...
-  ----completed in 0.688029 seconds---
+  ----completed in 0.986702 seconds---
+
   --Calculating shear centres...
-  ----completed in 0.000054 seconds---
+  ----completed in 0.000075 seconds---
+
   --Assembling shear deformation coefficients...
-  ----completed in 1.184013 seconds---
+  ----completed in 1.506942 seconds---
+
   --Assembling monosymmetry integrals...
-  ----completed in 0.784923 seconds---
+  ----completed in 1.298219 seconds---
 
   --Calculating plastic properties...
-  ----completed in 0.669841 seconds---
+  ----completed in 0.045805 seconds---
 
```

Comment:
Interesting to see all of these time comparisons. They are all longer times _except_ for the plastic properties, which is a huge improvement :)